homeassistant/components/onewire/switch.py

```diff
@@ -0,0 +1,130 @@
+"""Support for 1-Wire switches."""
```

Comment:
this should also be part of separate PR, the general approach is one platform or feature per PR

Reply:
Removed in 45701c486c6a7e04cbe0d681f6b7f3ffded82976